java/testing/TestCamera.java

```diff
@@ -0,0 +1,64 @@
+import app.f3d.F3D.*;
+
+public class TestCamera {
+
+  static {
+    if (System.getProperty("os.name").startsWith("Windows"))
```

Comment:
why not on other oses ?

Reply:
I remember it was needed for Windows only, the old tests were passing on Linux/macOS